Formula/ooniprobe.rb

```diff
@@ -152,36 +189,32 @@ def install
     (buildpath/"ooni/settings.ini").atomic_write <<-EOS.undent
       [directories]
       usr_share = #{share}/ooni
-      var_lib = #{var}/lib/ooni
+      var_lib = #{var}/run/ooni
+      etc = #{etc}/run/ooni
     EOS
 
     ENV.prepend_create_path "PYTHONPATH", libexec/"lib/python2.7/site-packages"
     system "python", *Language::Python.setup_install_args(libexec)
     bin.install Dir["#{libexec}/bin/*"]
-    bin.env_script_all_files(libexec/"bin", :PYTHONPATH => ENV["PYTHONPATH"])
+    bin.env_script_all_files(libexec/"bin", :PYTHONPATH => ENV["PYTHONPATH"], :PATH => ENV["PATH"])
```

Comment:
It's explained in the comment above. Basically without setting the PATH inside of the homebrew generated script that wraps the execution of the python script, PATH (when running the plist) doesn't contain the homebrew path of commands.